lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -164,6 +176,35 @@ def notify_long_elapsed_time_if_needed(token)
           Kuroko2.logger.info(message)
         end
       end
+
+      def auto_retry(node, token)
+        sleep_for_each_retry(node, token)
+
+        token.context['RETRY'][node.path]['retried_count'] += 1
+
+        message = "(token #{token.uuid}) Retry current node: '#{node.type}: #{node.option}'"
+        token.job_instance.logs.info(message)
+        Kuroko2.logger.info(message)
+
+        message = "(token #{token.uuid}) The number of retries: " << 
+          "#{token.context['RETRY'][node.path]['retried_count']} / #{token.context['RETRY'][node.path]['count']}"
+        token.job_instance.logs.info(message)
+        Kuroko2.logger.info(message)
+      end
+
+      def sleep_for_each_retry(node, token)
+        message = "(token #{token.uuid}) Sleep for #{token.context['RETRY'][node.path]['sleep_time']} seconds"
```

Comment:
After displaying the retry message, this sleep message should be displayed.

Reply:
I have modified it in 2c45746.